my_portfolio/resources/css/style.css

```diff
@@ -0,0 +1,733 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+header img {
+  width: 25px;
+  border-radius: 100%;
+  padding: 10px 0;
+}
+
+#home {
+  height: 93vh;
+  overflow: hidden;
+  align-items: center;
+  text-align: center;
+  background-color: #d3d2ff;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
```

Comment:
- I checked your css and I realised you  put in a great deal of work, bravo! it would be fair to point some observation so that the work will be even more pleasing
-  [ ] first I notice duplication of ```ids #``` and ```classes .``` e.g ```#home``` and ```#home`` not a good practice there a couple pls remove duplicates ensure you only have selectors
-  [ ] ordering the flow of the tags base on sections and elements eg
      - all nav sections and nav related selectors and style rule should be in one place
      - All home selectors in one place also and you should try to simulate a flow
      - All footer styles should be in one section
      - Good order is often a good practice, at the moment I am seeing different footer in between different sectiosn not being in one single portion in the design flow. kindly do something about that.

Reply:
Noted sir